test/functions/whereClauses/whereClauseParamArgMismatch3.good

```diff
@@ -1,2 +1,2 @@
 whereClauseParamArgMismatch3.chpl:10: error: unresolved call 'foo(1.2)'
-whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(8))
+whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(64))
```

Comment:
@bradcray I chose to update the .good to match the .chpl file, but I could instead update the .chpl file to use uint(8) to match whereClauseParamArgMismatch2's signature if you'd prefer